### ISSUE TITLE
Don't retriggers handler when seeking to the same frame

### ISSUE THREAD

**Issue by SBoudrias** [jeremyckahn/shifty#62]
When seeking through a tweenable, I think it would make sense to not triggers the step handlers if there's no change at the time head.

For example:

```js
tween.seek(15);
// This is triggering the step handler as expected.

// Then some time later
tween.seek(15);
// As we're still at the same position, we shouldn't triggers the handlers again
```

This is easily handled by user side code, but handling this on the side of shifty would make user code cleaner and probably helps to silently improve performances.

**Comment by jeremyckahn** [comment]
Good call, I'll get to this soon!

**Comment by SBoudrias** [comment]
Actually, this is not so easy to handle user side - even more in the setup we have on Yelp where we coordinates multiples tween based off a single one. So yeah, having this by default inside shifty would be very helpful!

**Comment by jeremyckahn** [comment]
This should be pretty straightforward to do.  I don't have time to implement and test this now, but hopefully in the next day or so.  If you need this sooner, feel free to make a Pull Request.

**Comment by jeremyckahn** [comment]
Okay, now I [actually referenced the correct issue](https://github.com/jeremyckahn/shifty/issues/63#issuecomment-74992835).  Please test this out when you can and let me know if it's working for you.

**Comment by jeremyckahn** [comment]
@SBoudrias, did you get a chance to test out this fix?

**Comment by SBoudrias** [comment]
Not yet, we have a bad infrastructure for open source third parties, so it's pretty involving. I should have some time to check it out next week.

**Comment by SBoudrias** [comment]
Hey @jeremyckahn, sorry this took so long! Finally got around to test it on Yelp and the fix is resolving the issue described here.

**Comment by jeremyckahn** [comment]
No problem.  I've merged the fix into `master`, and it is available as version [1.3.11](https://github.com/jeremyckahn/shifty/releases/tag/1.3.11).  Thanks for your help!
